spec/interactors/users/generate_access_token_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+describe Users::GenerateAccessToken do
+  describe ".call" do
+    let(:interactor) { described_class.new(user: user) }
+
+    context "when params are valid" do
+      let(:user) { create :user }
+
+      ENV["JWT_SECRET"] = "rspec"
```

Comment:
You need to add JWT_SECRET to CI env here: https://github.com/MaximStadnikov/task_tracker/blob/master/.github/workflows/action-tests.yml#L7